plugins/youtube_a.js

```diff
@@ -43,7 +43,7 @@ hoverZoomPlugins.push({
         }
 
         function prepareVideoPreview(link, id) {
-            if (link.hasClass('hoverZoomLoading') || link.hasClass('hoverZoomLink')) return;
+            f (link.hasClass('hoverZoomLoading') || link.hasClass('hoverZoomLink') || link.hasClass('ytp-title-link')) return;
```

Comment:
I don't think this line has correct syntax. Please test the code before sending pull requests.